src/components/tooltip/tooltip-theme.scss

```diff
@@ -1,6 +1,6 @@
-md-tooltip.md-THEME_NAME-theme {
-  color: '{{background-700-contrast}}';
-  .md-content {
-    background-color: '{{background-700}}';
+.md-tooltip.md-THEME_NAME-theme {
+  color: '{{background-A100}}';
```

Comment:
@crisbeto That is a good question. I am not sure, but the result was the same. I will have to look into why I changed them.